docs/source/community/build_ci_governance.rst

```diff
@@ -0,0 +1,20 @@
+PyTorch Governance | Build + CI
+===============================
+
+How to Add a New Maintainer
+---------------------------
+
+For the person to be a maintainer, a person needs to:
```

Comment:
Can we include a definition of what we think a maintainer is and what their responsibilities/privileges are?  The difference between a "maintainer" and a "contributor" seems important here.

Clearly, simply making any six commits isn't enough for a person to qualify. That's why we have a voting mechanism included